deployment/volumes/neo4j-data.yaml

```diff
@@ -9,4 +9,4 @@
       - ReadWriteOnce
     resources:
       requests:
-        storage: {{ .Values.neo4jStorage }}
+        storage: 5Gi
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-04-16T07:12:23Z" title="Thursday, April 16th 2020, 9:12:23 am +02:00">Apr 16, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -9,4 +9,4 @@
       - ReadWriteOnce
     resources:
       requests:
-        storage: {{ .Values.neo4jStorage }}
+        storage: 5Gi
```


But why we have changed this here if we don’t merge? @mattwr18 
We haven’t used it for our DevOps at all.